packages/normalizr/src/schemas/Polymorphic.js

```diff
@@ -56,6 +56,9 @@ export default class PolymorphicSchema {
   }
 
   denormalizeValue(value, unvisit) {
+    if (value === undefined) {
+      return [value, false, false];
+    }
```

Comment:
wouldn't this still crash if value is `null`?
```suggestion
    if (value == null) {
      return [value, false, false];
    }
```

Reply:
null is dealt with on line 101 of denormalize. added null to test for sanity

Reply:
this is a separate function that could be called independently, with `null`. in that case, it would crash.

Reply:
okay updating types so they can't be passed null